src/main.c

```diff
@@ -232,6 +240,9 @@ main(int argc, char **argv)
             case 'F':
                 o.out_file = optarg;
                 break;
+            case 'V':
+                print_version();
+                break;
```

Comment:
Nitpick(ish): The `break;` is unnecessary. In general I prefer to have them anyway but I think it's best to use the same approach for this and the `default` case where `print_usage()` does not have a `break` following it just to keep things consistent.